# Mutual Inductance and Transformers
### *Chapter 4 Section 7* -- AC Steady State Analysis

The previous analysis of a coil assumed that the only flux linking a coil was that due to its
own current, and consequently, the only voltage induced was that due to this current. In this
Chapter coupled coils and phenomenon of mutual inductance will be discussed. A two-port
equations expressing voltages by currents, both in time-domain and frequency-domain, will
be considered. Then, the most practical use of this phenomenon in transformers will be
studied. Basic transformer built of practical coils will be considered at first, next an ideal
transformer. Finally the ideal transformer based model of practical transformer will be
described.

## Mutual Inductance – Basic Transformer
Consider two coupled coils, as shown in Fig. 4.7.1.  

![Fig-4.7.1](Figures/Fig-4.7.1.png)

The current $i_1$ produces in coil 1 flux $\phi_{11}$. Part of this flux threads coil 2, the remainder is coil 1's leakage flux. They are denoted '$\phi_{21}$' and '$\phi_{l1}$' respectively.  Similarly, $i_2$ produces in coil 2 flux $\phi_{22}$ that is split into two fluxes, $\phi_{12}$ and $\phi_{l2}$.  

\begin{equation}\tag{1a}
	\phi_{11} = \phi_{21} + \phi_{l1}
\end{equation}

\begin{equation}\tag{1b}
	\phi_{22} = \phi_{12} + \phi_{l2}
\end{equation}

Electric analog of this magnetic circuit is presented in Fig. 4.7.2  

![Fig-4.7.2](Figures/Fig-4.7.2.png)

Via KCL,  

\begin{equation}\tag{2.1}
    \text{a: }\;\phi_{11} + \phi_{12} - \phi_{l1} = \phi \quad\quad\text{and}\quad\quad \text{b: }\;\phi = \phi_{22} + \phi_{21} - \phi_{l2}
\end{equation}

Via the above two equations and equivalance principle,  

$$ \phi_{11} + \phi_{12} - \phi_{l1} = \phi_{22} + \phi_{21} - \phi_{l2}$$

Via the above equation and equations (1a,b),  

$$ \phi_{21} + \phi_{12} = \phi_{12} + \phi_{21}$$

The electric analog in figure 4.7.2 is thus consistent with equations (1a,b).

Via KVL and Ohm's Law,  

\begin{equation}\tag{2.2}
    \text{a: }\;U_{l1} = \phi_{l1}R_{l1} = i_1z_1 \quad\quad \text{b: }\; U_m = R_m\phi = i_1z_1 + i_2z_2 \quad\quad\text{c: }\; i_2z_2 = \phi_{l2}R_{l2} = U_{l2}
\end{equation}

And, via equations (2.1a) and (1b), or equations (2.1b) and (1a),    

\begin{equation}\tag{2.3}
    \phi = \phi_{11} + \phi_{22} - \phi_{l1} - \phi_{l2}
\end{equation}

where $R_{l1}$ and $R_{l2}$ are the magnetic resistances of the leakages, and $R_m$ is a core magnetic resistance.  

Superposition principle can be applied and a circuit of Fig. 4.7.2 can be split into two
subcircuits, as presented in Fig. 4.7.2a.  

![Fig-4.7.2a](Figures/Fig-4.7.2a.png)

\begin{equation}\tag{2a.1}
    i_1z_1 = \sideset{_{1\,}}{_m}U = R_m\phi_{21} = R_{l1}\phi_{l1}
\end{equation}

\begin{equation}\tag{2a.2}
    i_2z_2 = \sideset{_{2\,}}{_m}U = R_m\phi_{12} = R_{l2}\phi_{l2}
\end{equation}

Via the above two equations and superposition principle,  

$$ U_m = \sideset{_{1\,}}{_m}U + \sideset{_{2\,}}{_m}U$$

The total flux threading coil 1 is the sum of two components:  

\begin{equation}\tag{2a}
	\phi_{t1} = z_1\phi_1 = z_1\phi_{11}\pm z_1\phi_{12} = z_1\left(\frac{i_1z_1}{R_{l1}} + \frac{i_1z_1}{R_m}\right) \pm z_1\frac{i_2z_2}{R_m} = L_1i_1 \pm Mi_2
\end{equation}

The total flux threading coil 2 is the sum of two components:  

\begin{equation}\tag{2b}
	\phi_{t2} = z_2\phi_2 = z_2\phi_{22}\pm z_2\phi_{21} = z_2\left(\frac{i_2z_2}{R_{l2}} + \frac{i_2z_2}{R_m}\right) \pm z_2\frac{i_1z_1}{R_m} = L_2i_2 \pm Mi_1
\end{equation}

The proof for equation (2a) is:  

$$ \phi_{t1} = z_1\phi_1 \quad\quad\text{where }\; \phi_1 = \phi_{11} \pm \phi_{12}$$

Via equations (1a) and (2a.1),  

$$ \phi_{11} = \phi_{21} + \phi_{l1} = \frac{i_1z_1}{R_m} + \frac{i_1z_1}{R_{l1}}$$

And, via equation (2a.2),  

$$ \phi_{12} = \frac{i_2z_2}{R_m}$$

Via the above three equations,  

$$ \phi_{t1} = z_1\left(\frac{i_1z_1}{R_{l1}} + \frac{i_1z_1}{R_m}\right) \pm z_1\frac{i_2z_2}{R_m}$$

Based on equation (2a),  

\begin{equation}\tag{2a.3}
    \text{a: }\;L_1 = z_1^2\frac{R_m + R_{l1}}{R_{l1}R_m} \quad\quad\text{and}\quad\quad \text{b: }\;M = \frac{z_1z_2}{R_m}
\end{equation}

And, based on equation (2b),  

\begin{equation}\tag{2a.3c}
    L_2 = z_2^2\frac{R_m + R_{l2}}{R_{l2}R_m}
\end{equation}

Constant of proportionality between one coil current and a flux that is produces in the coupled
coil is called the **mutual inductance** $M$. This constant can be expressed by self inductances of individual coils.  

\begin{equation}\tag{3}
	M = k\sqrt{L_1L_2}
\end{equation}

where, $0\le k \le 1$ is the **coefficient of coupling**, a measure of the degree to which the flux
produced by one coil threads another. If there is no coupling then, $k=0\;\implies\;M=0$ . For
tightly coupled coils, which is the most desirable situation, $k=1\;\implies\;M=\sqrt{L_1L_2}$. Via equation 
(2), two-port equations can be obtained.  

\begin{equation}\tag{4a}
    u_1 = \frac{\mathrm{d}\phi_{t1}}{\mathrm{d}t} = L_1i_1'(t)\pm Mi_2'(t)
\end{equation}

\begin{equation}\tag{4b}
    u_2 = \frac{\mathrm{d}\phi_{t2}}{\mathrm{d}t} = L_2i_2'(t)\pm Mi_1'(t)
\end{equation}

The plus or minus is used because the fluxes produced by the coupled coils may point in the same or opposite direction.  In the case of the magnetic circuit shown in figure 4.7.1, the fluxes point in the same direction and thus the plus sign is used in the appropriate equations.

Via equations (2a.3a,c),  

$$ L_1L_2 = z_1^2z_2^2\frac{R_m^2 + R_m(R_{l1} + R_{l2}) + R_{l1}R_{l2}}{R_{l1}R_{l2}R_m^2}$$

And, via the above equation and equations (2a.3b) and (3),  

\begin{equation}\tag{2a.4}
    k = \sqrt{\frac{R_{l1}R_{l2}}{(R_m+R_{l1})(R_m+R_{l2})}}
\end{equation}

### Dot convention
Currents entering the dotted ends are creating additive fluxes. Dotted ends have a positive
voltage at the same time.  

A circuit symbol of coupled coils is presented in Fig. 4.7.3.  

![Fig-4.7.3](Figures/Fig-4.7.3.png)

Using the phasor notation, coupled coils equations are:  

\begin{equation}\tag{5a}
    U_1(j\omega) = j\omega L_1I_1(j\omega) \pm j\omega MI_2(j\omega)
\end{equation}

\begin{equation}\tag{5b}
    U_2(j\omega) = j\omega L_2I_2(j\omega)\pm j\omega MI_1(j\omega)
\end{equation}

#### Example 4.7.1
Consider two practical coupled coils connected in series, as shown in Fig. 4.7.4. Find the
circuit equivalent.  

![Fig-4.7.4](Figures/Fig-4.7.4.svg)

$$ U_1(j\omega) = R_1I_1(j\omega) + j\omega L_1I_1(j\omega) + j\omega MI_2(j\omega)$$

$$ U_2(j\omega) = R_I2(j\omega) + j\omega L_2I_2(j\omega) + j\omega MI_1(j\omega)$$

Via the fact that $I(j\omega) = I_1(j\omega) = I_2(j\omega)$, $U(j\omega) = U_1(j\omega)+U_2(j\omega)$, and the above two equations,  

\begin{equation}\tag{6a}
    U(j\omega) = I(j\omega)\big(R_1 + R_2 + j\omega(L_1+L_2 + 2M)\big)
\end{equation}

\begin{equation}\tag{6b}
    U(j\omega) = I(j\omega)(R_s + j\omega L_s)
\end{equation}

Thus,  

\begin{equation}\tag{7}
    \text{a: }\;L_s = L_1+L_2+2M \quad\quad\text{and}\quad\quad \text{b: }\;R_s = R_1+R_2
\end{equation}

The equivalent series inductance of two coupled coils is sum of self inductances plus the
doubled mutual inductance. For negative coupling the doubled mutual inductance should be
subtracted.

#### Example 4.7.2
A coil of $z=100$ turns connected to a $U=100\mathrm{V}$, $\omega=1000r\,\mathrm{rad}/\mathrm{s}$ supply has the following parameters: $L=1\mathrm{H}$, $R=10\Omega$ . Consider a short-circuit of one turn. Study the effect of such failure.  

A coil with one turn shorted can be considered as pair of coupled coils, as presented in Fig.
4.7.5.  

![Fig-4.7.5](Figures/Fig-4.7.5.png)

Via equation (3.1.14),  

$$ R_m = \frac{z^2}{L} = 10\mathrm{k}\Omega $$

$$ L_1 = \frac{99^2}{R_m}\cdot0.99 = 970.299\mathrm{mH} \quad\quad\text{and}\quad\quad R_1 = 9.9\Omega$$

In [1]:
z = 100; L = 1; R = 10; U = 100; omega = 1000
R_m = z**2/L
L_1 = 0.99*99**2/R_m; R_1=9.9
L_1

0.970299

$$ L_2 = \frac{1}{R_m}\cdot0.01 = 0.1\mathrm{mH} \quad\quad\text{and}\quad\quad R_2 = 0.1\Omega$$

In [2]:
L_2 = 1/R_m; R_2 = 0.1
L_2

0.0001

**Prior** to the short circuit:  

$$ U(j\omega) = RI(j\omega) + j\omega LI(j\omega) \quad\implies\quad I(j\omega) = \frac{U}{R+j\omega L} $$

$$ I(\omega) = \frac{U}{\sqrt{R^2+(\omega L)^2}} \approxeq 99.995\mathrm{mA}$$

In [3]:
from math import sqrt

In [4]:
I = U/sqrt(R**2 + (omega*L)**2)
I

0.09999500037496875

$$ \alpha_i = -\arccos \frac{R}{\sqrt{R^2+(\omega L)^2}} \approxeq -89.43^{\circ}$$

In [5]:
import numpy as np
import matplotlib.pyplot as plt

In [6]:
alpha_i = np.arccos(R / sqrt(R**2+(omega*L)**2))
np.degrees(alpha_i)

89.42706130231652

The dissipated power is:  

\begin{equation}
    P = I^2(\omega)R = \frac{U^2R}{R^2 + (\omega L)^2} = \frac{10^5}{100 + 10^6}\,\mathrm{W} \approxeq 0.1\mathrm{W}
\end{equation}

In [7]:
P = 10**5/(100+10**6)
P

0.0999900009999

**After** the short circuit:  

\begin{equation}\tag{2.1}
    U(j\omega) = R_1I_1(j\omega) + j\omega L_1I_1(j\omega) + j\omega MI_2(j\omega)
\end{equation}

$$ 0 = R_2I_2(j\omega) + j\omega L_2I_2(j\omega) + j\omega MI_1(j\omega)$$

Assuming a coupling coefficient of $k=1$, i.e. assuming no flux leakage,  

\begin{equation}\tag{2.4}
    M = \sqrt{L_1L_2} \approxeq 9.85 \mathrm{mH}
\end{equation}

In [8]:
M = sqrt(L_1*L_2)
M

0.009850375627355537

\begin{equation}\tag{2.2}
    I_2(j\omega) = -\frac{j\omega MI_1(j\omega)}{R_2 + j\omega L_2}
\end{equation}

Via equations (2.1,2),  

$$ U(j\omega) = I_1(j\omega)\left(R_1 + j\omega L_1 + \frac{(\omega M)^2}{R_2 + j\omega L_2}\right) = Z(j\omega)I(j\omega)$$

The impedance is given by:  

$$ Z(j\omega) = R_1 + \frac{(\omega M)^2}{R_2^2 + (\omega L_2)^2} + j\omega L_1 - j\frac{\omega L_2}{R_2^2 + (\omega L_2)^2}$$

\begin{equation}\tag{2.3}
    = R_1 + \frac{(\omega M)^2R_2}{R_2^2 + (\omega L_2)^2} + j\omega\left(L_1 - \frac{(\omega M)^2L_2}{R_2^2 + (\omega L_2)^2}\right)
\end{equation}

In [9]:
Z = complex(R_1 + (omega*M)**2*R_2/(R_2**2 + (omega*L_2)**2), \
            omega*L_1 - omega*(omega*M)**2*L_2/(R_2**2 +(omega*L_2)**2))
Z

(495.04949999999997+485.1495j)

In [10]:
sqrt(Z.real**2 + Z.imag**2)

693.1407106789356

\begin{equation}\tag{2.3a}
    Z(\omega) \approxeq 693.14\Omega
\end{equation}

Finally,  

\begin{equation}\tag{2.4}
    I(\omega) = \frac{U}{Z(\omega)} \approxeq 144.27\mathrm{mA}
\end{equation}

In [11]:
I_rms = U/sqrt(Z.real**2 + Z.imag**2)
I_rms

0.14427085072243034

Finally, the dissipated power is given by:  

\begin{equation}\tag{2.5}
    P = I^2(\omega)\mathbf{Z}_\mathfrak{Re} \approxeq 10.3\mathrm{W}
\end{equation}

In [12]:
I_rms**2*Z.real

10.303999089125245

The impedance magnitude practically has not changed, and consequently rms current remains
unchanged, however the resistance and then dissipated power has increased 100 times !!! This
causes rapid increase of temperature, an isolation melts and next turns are shorted, what
completely destroys (burns) the coil.

**Basic transformer**, an air-core (liear) transformer built of two coupled coils is considered
next.  

A transformer is an electronic device that uses magnetically coupled coils to transfer energy
from one circuit to another. This device has two ports available for connection to external
circuitry. One of this ports is called the **primary port** and usually it is connected to an
external source. The other port is called the **secondary port** and usually it is connected to a
load. Then, terms primary/secondary circuit, winding, voltage or current are used. The
primary circuit has been simplified to its Thevenin equivalent, the load has been reduced to its
equivalent impedance. It should be emphasized, that the primary and the secondary circuit are
electrically isolated. Basic transformer circuit described in phasor domain is presented in Fig.
4.7.6. For simplicity of further derivations, it has been assumed that coils are resistanceless.  

![Fig-4.7.6](Figures/Fig-4.7.6.png)

Transformer itself is described in phasor domain by equations (4.7.4b). The secondary circuit
(load) is described by the following equation  

\begin{equation}\tag{9}
    U_2(j\omega) = -Z_l(j\omega)I_2(j\omega)
\end{equation}

\begin{equation}\tag{9a}
    U_2(j\omega) = j\omega L_2I_2(j\omega) + j\omega MI_1(j\omega)
\end{equation}

Via equations (9) and (9a),  

$$ j\omega MI_1(j\omega) = -I_2(j\omega)\big(j\omega L_2 + Z_l(j\omega)\big)$$

$$\frac{I_1(j\omega)}{I_2(j\omega)} = -\frac{j\omega L_2 + Z_l(j\omega)}{j\omega M}$$

\begin{equation}\tag{M1}
    \frac{I_2(j\omega)}{I_1(j\omega)} = -\frac{j\omega M}{j\omega L_2 + Z_l(j\omega)}
\end{equation}

The load current and voltage do not satisfy the passive sign convention, and, for that reason, the
minus sign appears at the right side of equation (4.7.9). It is interesting to study the
impedance looking into the primary port.  

\begin{equation}\tag{10}
    Z_1(j\omega)=\frac{U_1(j\omega)}{I_1(j\omega)}
\end{equation}

\begin{equation}\tag{M2}
    U_1(j\omega)=E_\mathrm{o}(j\omega) - Z_\mathrm{t}(j\omega)I_1(j\omega) = j\omega L_1I_1(j\omega) + j\omega MI_2(j\omega)
\end{equation}

$$ I_1(j\omega)\big(Z_\mathrm{t}(j\omega) + j\omega L_1\big) = E_\mathrm{o}(j\omega) - j\omega MI_2(j\omega)$$

Via equations (10) and (M2),  

$$ Z_1(j\omega) = j\omega L_1 + j\omega M \,\frac{I_2(j\omega)}{I_1(j\omega)}$$

And, via the above equation and equation (M1),  

\begin{equation}\tag{10a}
    Z_1(j\omega) = j\omega L_1 + \frac{(\omega M)^2}{j\omega L_2 + Z_l(j\omega)}
\end{equation}

The first term depends on the primary coil while the second term is due to the coupling, so
called the **reflected impedance**.  

\begin{equation}\tag{10b}
    Z_r(j\omega) = \frac{(\omega M)^2}{j\omega L_2 + Z_l(j\omega)} = \frac{(\omega M)^2}{R_l + j\big(\omega L_2 + X_l(\omega)\big)}  = R_r + jX_r(\omega)
\end{equation}

Let $Z_l(j\omega):=\mathbf{Z_l}$, which is consistent with the phasor and complex number notation used throughout this notebook.  

$$ Z_r(j\omega) = \frac{(\omega M)^2}{\mathfrak{Re}\{\mathbf{Z_l}\} + j\big(\omega L_2 + \mathfrak{Im}\{\mathbf{Z_l}\}\big)} = (\omega M)^2\,\frac{R_l - j\big(\omega L_2 + X_l(\omega)\big)}{R_l^2 + \big(\omega L_2 + X_l(\omega) \big)^2}$$

Then, the transformer together with its load can be replaced by series connection of two
impedances. The obtained one-loop circuit, presented in Fig. 4.7.7, is often used to simplify
analysis of the basic transformer circuit of Fig. 4.7.6.  

![Fig-4.7.7](Figures/Fig-4.7.7.png)

Then, the transformer together with its load can be replaced by series connection of two
impedances. The obtained one-loop circuit, presented in Fig. 4.7.7, is often used to simplify
analysis of the basic transformer circuit of Fig. 4.7.6.

## Ideal Transformer
A transformer scales, or transforms, the voltage, current and impedance levels of the circuit.
The secondary-to-primary current and voltage ratios will be discussed for an ideal transformer
that may be thought of as the first-order model of an iron-core transformer (complex model
will be discussed in the next section of this Chapter). An **ideal transformer** satisfies the
following three assumptions.

1. Windings are resistanceless:  $R_1=R_2=0$.
2. Leakage fluxes are equal to zero:  $\phi_{l1}=\phi_{l2}=0\;\implies\;k=1$.
3. The core magnetic material has unboundedly large permeability $\mu_r$, implying that the reluctance or magnetic resistance $R_m$ of the core is equal to zero, implying that the primary and secondary inductance, denoted as '$L_1$' and '$L_2$' respectively, are also unboundedly large;  Their ratio, however, is finite.  

$$ \mu_r = \infty \quad\implies\quad R_m = 0\quad\implies\quad L_1,L_2=\infty$$  
    \begin{equation}\tag{14}
        \frac{L_2}{L_1} = \left(\frac{z_2}{z_1}\right)^2 = n^2\ne\infty
    \end{equation}
    Note that the above equation follows from equations (2a,b).


Via equations (2) and the three assumption of an ideal transformer,  

$$ z_1\phi_1 = z_1\phi = \frac{z_1^2i_1 \pm z_1z_2i_2}{R_m} = L_1i_1 \pm \sqrt{L_1L_2}i_2$$

and  

$$ z_2\phi_2 = z_2\phi= \frac{z_2^2i_2 \pm z_1z_2i_1}{R_m} = L_2i_2 \pm \sqrt{L_1L_2}i_1$$

Via the above two equations and equivalence principle,  

$$ \frac{L_1i_1 \pm \sqrt{L_1L_2}i_2}{z_1^2i_1 \pm z_1z_2i_2} = \frac{L_2i_2 \pm \sqrt{L_1L_2}i_1}{z_2^2i_2 \pm z_1z_2i_1}$$

$$ \frac{L_1i_1 \pm \sqrt{L_1L_2}i_2}{z_1} = \frac{L_2i_2 \pm \sqrt{L_1L_2}i_1}{z_2} $$

Which implies that:  

$$ \frac{z_2}{z_1}\left(i_1\pm \sqrt{\frac{L_2}{L_1}}\,i_2 \right) = \frac{L_2}{L_1}\,i_2 \pm \sqrt{\frac{L_2}{L_1}}\,i_1$$

$$ \frac{z_2}{z_1}\,\frac{i_1\sqrt{L_1}\pm i_2\sqrt{L_2}}{\sqrt{L_1}} = \frac{L_2i_2 \pm \sqrt{L_1L_2}i_1}{L_1}$$

$$ \frac{z_2}{z_1} = \frac{L_2i_2 \pm i_1\sqrt{L_1L_2}}{L_1i_1 \pm i_2\sqrt{L_1L_2}}$$

Squaring both sides of the above equations gives:  

$$ \left(\frac{z_2}{z_1}\right)^2 = \frac{(L_2i_2)^2 \pm 2i_1i_2L_2\sqrt{L_1L_2} + i_1^2L_1L_2}{(L_1i_1)^2 \pm 2i_1i_2L_1\sqrt{L_1L_2} + i_2^2L_1L_2}$$

$$ = \frac{L_2}{L_1}\,\frac{L_2i_2^2 \pm 2i_1i_2M + i_1^2L_1}{L_1i_1^2 \pm 2i_1i_2M + i_2^2L_2} = \frac{L_2}{L_1}$$

Q.E.D.

An ideal transformer electric analog is presented in Fig. 4.7.8.  

![Fig-4.7.8](Figures/Fig-4.7.8.png)

Whereas windings are resistanceless (assumption 1) and same flux flows through both coils
(assumption 2), therefore their voltages are:  

\begin{equation}\tag{11}
    u_1 = z_1\,\frac{\mathrm{d}\phi}{\mathrm{d}t} \quad\quad\text{and}\quad\quad u_2 = z_2\,\frac{\mathrm{d}\phi}{\mathrm{d}t}
\end{equation}

As the core magnetic resistance (voltage) is zero (assumption 3), then the currents are related
by the following KVL equation:  

\begin{equation}\tag{12}
     i_1z_1 + i_2z_2 = R_m\phi = u_m = 0
\end{equation}

Via equations (11) amd (12), secondary-to-primary current and voltage ratios are:  

\begin{equation}\tag{13}
    \text{a: }\;\frac{u_2}{u_1} = \frac{z_2}{z_1} = n \quad\quad\text{and}\quad\quad \text{b: }\;\frac{i_2}{i_1} = -\frac{z_1}{z_2} = -\frac{1}{n}
\end{equation}

where,  

\begin{equation}\tag{14}
    n = \frac{z_2}{z_1}
\end{equation}

is the only parameter that characterizes an ideal transformer, so called **turns ratio**. Its value
defines the transformer character:  

* $n>1$:  step-up transformer.
* $n<1$:  step-down transformer.
* $n=1$:  isolating transformer.

It is worth to observe, that an ideal transformer secondary-to-primary current and voltage
ratios are fixed, independent of load and frequency. An ideal transformer phasor equations are:  

\begin{equation}\tag{15}
    \text{a: }\;U_2(j\omega) = n\,U_1(j\omega)\quad\quad\text{and}\quad\quad\text{b: }\;I_2(j\omega) = -\frac{1}{n}\,I_1(j\omega)
\end{equation}

The circuit symbol for an ideal transformer is similar to that for nonideal transformer (Fig.
4.6.6) except that the turns ratio is specified rather than inductances (all three are infinitely
large) and pair of parallel lines is drawn. An ideal transformer with primary source and
secondary load is presented in Fig. 4.7.9.  

![Fig-4.7.9](Figures/Fig-4.7.9.png)

$$ U_2(j\omega) = -Z_l(j\omega)I_2(j\omega) = Z_l(j\omega)I_1(j\omega)\,\frac{1}{n}$$

$$ n\,U_1(j\omega) = Z_l(j\omega)I_1(j\omega)\,\frac{1}{n}$$

$$ Z_1(j\omega) = \frac{U_1(j\omega)}{I_1(j\omega)}$$

Via the above two equations, impedance looking into the primary port is:  

\begin{equation}\tag{16}
    Z_1(j\omega) = Z_l(j\omega)\,\frac{1}{n^2}
\end{equation}

Thus, an ideal transformer together with its secondary load $Z_l(j\omega)$ is equivalent to an
impedance of $Z_1(j\omega)$ reflected into the primary circuit. Then, apart of previously
discussed uses to isolate two circuits and step-up or step down a voltage level, an ideal
transformer can be used to impedance scale a load such that maximum power transfer
condition is achieved. The latter use will be illustrated in the next example.

### Example 4.7.3
An audio amplifier produces the Thevenin equivalent voltage phasor $E_\mathrm{o}(j\omega)$ through an impedance (resistance) of $Z_\mathrm{t}(j\omega) = R_\mathrm{t}=72\Omega$.  The produced power is to be delivered to a load speaker with an impedance (resistance) of $Z_l(j\omega)=R_l=8\Omega$ through an ideal transformer.  Determination of the turns ratio, denoted as '$n$', to maximize the power supplied by the source (or, the power absorbed by the speaker) is the task.  The amplifier-transformer-speaker circuit is presented in figure 4.7.9.  

For the maximum power transfer, the load (speaker) impedance $Z_l(j\omega)$ seen by the source (i.e. the impedance reflected into the primary circuit of the amplifier, denoted as '$Z_1(j\omega)$') should be equal to the complex conjugate of the source impedance, which is purely real or resistive;  $Z_\mathrm{t}(j\omega) = R_\mathrm{t} = 72\Omega$.  Thus, for the maximum power supplied to the speaker from the amplifier,  

\begin{equation}\tag{17}
	Z_1(j\omega) = \frac{1}{n^2}\,R_l = R_\mathrm{t}\quad\implies\quad n = \sqrt{\frac{R_l}{R_\mathrm{t}}} = \frac{1}{3}
\end{equation}

In [13]:
72/8

9.0

Which means that a $1\!:\!3$ step-down transformer should be used to impedance match the source to the give load (the speaker, in this case).

In some practical applications, transformer is replaced by an **autotransformer**.
Autotransformer is built of a single winding with the **tap point**. Then, it can be considered as
two coupled coils connected in series. Primary circuit is connected between the tap point and
one terminal of the winding, secondary circuit is connected to winding terminals, as presented
in Fig. 4.7.10a. Same effect can be obtained by series connection of two-winding transformer
coils, as presented in Fig. 4.7.10b.  

![Fig-4.7.10a](Figures/Fig-4.7.10a.png)

![Fig-4.7.10b](Figures/Fig-4.7.10b.png)

Via equation (12) and the fact that both $I_1(j\omega)$ and $I_2(j\omega)$ flow through $z_1$,  

\begin{equation}\tag{3.3}
    z_1(i_1 + i_2) = -z_2i_2
\end{equation}

$$ i_1 = -ni_2 - i_2 = -i_2(n+1) \quad\implies\quad I_1(j\omega) = -I_2(j\omega)(n+1)$$

And, via equation (2a,b), the assumptions that an ideal (auto)transformer satisfies, and the fact that both $i_1$ and $i_2$ flow through $z_1$,  


$$ z_1\phi_1 = z_1\phi = z_1\phi_{11} \pm z_1\phi_{12} = (i_1+i_2)\,\frac{z_1^2}{R_m} \pm i_2z_1\,\frac{z_2}{R_m} = L_1(i_1+i_2) \pm Mi_2$$

\begin{equation}\tag{3.1}
    z_1\phi_1 = z_1\phi= \frac{(i_1+i_2)z_1^2 \pm i_2z_1z_2}{R_m}
\end{equation}

$$ z_2\phi_2 = z_2\phi_{22}\pm z_2\phi_{21} = i_2\,\frac{z_2^2}{R_m} \pm (i_1+i_2)z_2\,\frac{z_1}{R_m} = L_2i_2 \pm M(i_1+i_2)$$

\begin{equation}\tag{3.2}
    z_2\phi_2 = z_2\phi=\frac{i_2z_1^2 \pm (i_1+i_2)z_1z_2}{R_m}
\end{equation}

Differentiating throughout equations (3.1) and (3.2) with respect to time gives:  

\begin{equation}\tag{3.1a}
    u_1 = z_1\,\frac{\mathrm{d}\phi}{\mathrm{d}t} 
\end{equation}

\begin{equation}\tag{3.2a}
    u_2 = z_2\,\frac{\mathrm{d}\phi}{\mathrm{d}t} + u_1
\end{equation}

Via equations (3.2a) and (3.1a) and equivalence principle,  

$$ u_2 = u_1n + u_1 = u_1(n+1) \quad\implies\quad U_2(j\omega) = U_1(j\omega)(n+1)$$

Thus, the turns ratio is:  

$$ n_a = \frac{z_2}{z_1} + 1 = \frac{z_2 + z_1}{z_1}$$

The impedance looking into the primary port, denoted as '$Z_1(j\omega)$', is given by:  

$$ Z_1(j\omega) = \frac{U_1(j\omega)}{I_1(j\omega)} \quad\quad\text{and}\quad\quad U_1(j\omega) = \frac{U_2(j\omega)}{n+1}$$

and,  

$$ U_2(j\omega) = -Z_l(j\omega)I_2(j\omega) \quad\implies\quad U_2(j\omega) = Z_l(j\omega)I_1(j\omega)\,\frac{1}{n+1}$$

Finally,  

$$ Z_1(j\omega) = \frac{Z_l(j\omega)}{(n+1)^2}$$

It should be emphasized, that an autotransformer couples primary and secondary circuit both
electrically and magnetically (no electrical isolation between circuits).

## Practical Iron-Core Transformer

In a practical iron-core transformer all three nonidealities: leakage fluxes, winding losses and
nonideality of magnetic material have to be taken into account. One model of a practical
transformer has been already discussed. The other, commonly used model consists of an ideal
transformer supplemented by elements representing nonidealities. Leakage fluxes and
winding losses are taken into account by series connection of the leakage inductance $L_{l1}$ ( $L_{l2}$ ) and winding resistance $R_1$ ($R_2$ ) in series with the primary (secondary) winding of an ideal transformer. Nonideality or practicality of a magnetic material (non-infinite inductancesMLL ,, 21 ) is taken into account by parallel connection of inductance $L_m$ (magnetization inductance of a core) with the primary winding. The obtained practical transformer circuit is presented in Fig. 4.7.11.  

![Fig-4.7.11](Figures/Fig-4.7.11.png)

In the model *without* the supplemental coils representing nonidealities (practicalities), via differentiating throughout equations (2a,b) with respect to time,  

$$ u_1 = i_1'z_1^2\,\left(\frac{1}{R_{l1}} + \frac{1}{R_m}\right) + i_2'z_1\,\frac{z_2}{R_m} + R_1i_1$$  

\begin{equation}\tag{M3a}
    = i_1'z_1^2\,\frac{R_m + R_{l1}}{R_{l1}R_m} + i_2'z_1\,\frac{z_2}{R_m} + R_1i_1
\end{equation}

and

 $$u_2 = i_2'z_2^2\,\left(\frac{1}{R_{l2}} + \frac{1}{R_m}\right) + i_1'z_2\,\frac{z_1}{R_m} + R_2i_2$$

\begin{equation}\tag{M4a}
    = i_2'z_2^2\,\frac{R_m + R_{l2}}{R_{l2}R_m} + i_1'z_2\,\frac{z_1}{R_m} + R_2i_2
\end{equation}

In the frequency domain:  

\begin{equation}\tag{M3b}
    U_1(j\omega) = j\omega L_1I_1(j\omega) + j\omega MI_2(j\omega) + R_1I_1(j\omega)
\end{equation}

and,  

\begin{equation}\tag{M4b}
    U_2(j\omega) = j\omega L_2I_2(j\omega) + j\omega MI_1(j\omega) + R_2I_2(j\omega)
\end{equation}

where,  

\begin{equation}\tag{M5}
    \text{a: }\;L_1 = z_1^2\,\frac{R_m+R_{l1}}{R_{l1}R_m}\quad\quad\text{b: }\;L_2 = z_2^2\frac{R_m+R_{l2}}{R_{l2}R_m}\quad\quad\text{c: }\;M = \frac{z_1z_2}{R_m}
\end{equation}

In the model *with* the supplemental coils, via KVL, element equations, and differentiating throughout equations (2a,b) with respect to time,  

\begin{equation}\tag{M6}
    \text{a: }\;u_1 = R_1i_1 + L_{l1}i_1' + L_mi_{1a}' \quad\quad\text{and}\quad\quad \text{b: }\;L_mi_{1a}' = u_1^* = L_1i_{1b}' + Mi_2' = z_1\phi'
\end{equation}

\begin{equation}\tag{M7}
    \text{a: }\;u_2 = R_2i_2 + L_{l2}i_2' + L_2i_2' + Mi_{1b}' \quad\quad\text{and}\quad\quad \text{b: }\;u_2^* = z_2\phi' = L_2i_2' + M_{1b}'
\end{equation}

Via equations (M6b) and (M7b) and equivalence principle,  

\begin{equation}\tag{M13}
    \frac{u_1^*}{z_1} = \frac{u_2^*}{z_2}
\end{equation}

\begin{equation}\tag{M8}
    \text{a: }\;u_1 = R_1i_1 + L_{l1}i_1' + u_1^* \quad\quad\text{and}\quad\quad \text{b: }\;u_2 = R_2i_2+ L_{l2}i_2' + nu_1^*
\end{equation}

Via KCL,  $i_1 = i_{1a} + i_{1b}$.  

$$ L_mi_{1a}' = L_1\big(i_1' - i_{1a}'\big) +Mi_2'$$

$$ i_{1a}'(L_m + L_1) = L_1i_1' + Mi_2' \quad\implies\quad i_{1a}' = \frac{L_1i_1' + Mi_2'}{L_m + L_1}$$

\begin{equation}\tag{M9}
    u_1^* = L_m\,\frac{L_1i_1' + Mi_2'}{L_m + L_1}
\end{equation}

Via equations (M9) and (M8a,b),  

$$ u_1 = R_1i_1 + L_{l1}i_1' + L_m\,\frac{L_1i_1' + Mi_2'}{L_m + L_1} \quad\quad \text{and}\quad\quad u_2 = R_2i_2 + L_{l2}i_2' + nL_m\,\frac{L_1i_1' + Mi_2'}{L_m + L_1}$$

Recall that, via equation (2a) and the assumptions that an ideal transformer makes,  

$$ L_1 = \frac{z_1^2}{R_m} \quad\quad\text{and}\quad\quad M = \frac{z_1z_2}{R_m}$$

Via the above two equations and equivalence principle,  

$$ \frac{z_1^2}{L_1} = \frac{z_1z_2}{M} \quad\implies\quad M = \frac{L_1z_2}{z_1} = L_1n$$

Thus,  

\begin{equation}\tag{M10}
    \text{a: }\;u_1 = R_1i_1 + L_{l1}i_1' + L_mL_1\,\frac{i_1' + ni_2'}{L_m + L_1} \quad\quad\text{and}\quad\quad \text{b: }\;u_2 = R_2i_2 + L_{l2}i_2' + nL_mL_1\,\frac{i_1' + ni_2'}{L_m+L_1}
\end{equation}

Via the above two equations, equation (2a), and the assumptions that an ideal transformer satisfies,  

\begin{equation}\tag{M11}
    \text{a: }\;u_1 = R_1i_1 + L_{l1}i_1' + L_m\big(i_1' + ni_2'\big) \quad\quad\text{and}\quad\quad \text{b: }\; u_2 = R_2i_2 + L_{l2}i_2' + nL_m\big(i_1' + ni_2'\big)
\end{equation}

Now, considering *both* models, via the above two equations and equations (M3a) and (M4a),  

$$ L_{l1}i_1' + L_m\big(i_1' + ni_2'\big) \cong i_1'z_1^2\,\left(\frac{1}{R_{l1}} + \frac{1}{R_m}\right) + i_2'z_1\,\frac{z_2}{R_m}$$

$$ L_{l1}i_1' \;\hat{=}\; \frac{i_1'z_1^2}{R_{l1}} \quad\implies\quad L_{l1} = \frac{z_1^2}{R_{l1}} \quad\quad\text{and}\quad\quad L_mi_1' \;\hat{=}\; \frac{i_1' z_1^2}{R_m} \quad\implies\quad L_m = \frac{z_1^2}{R_m}$$

Via equivalence principle,  

$$ L_{l1}R_{l1} = L_mR_m$$

$$ nL_mi_2' \;\hat{=}\; i_2'\,\frac{z_1z_2}{R_m}\quad\implies\quad nL_m = \frac{z_1z_2}{R_m}$$

$$ L_{l2}i_2' + nL_m\big(i_1' + ni_2'\big)\cong i_2'z_2^2\,\left(\frac{1}{R_{l2}} + \frac{1}{R_m}\right) + i_1'z_2\,\frac{z_1}{R_m}$$

$$ L_{l2} = \frac{z_2^2}{R_{l2}} \quad\quad\text{and}\quad\quad n^2L_m = \frac{z_2^2}{R_m} \quad\quad\text{and}\quad\quad nL_m = \frac{z_1z_2}{R_m}$$

Via the above three equations,  

$$ L_{l2}R_{l2} = n^2L_mR_m$$

Via differentiating throughout the equaton obtain by KCL and equation (M6b),  

$$ i_{1b}' = i_1' - \frac{L_1i_1' + Mi_2'}{L_m + L_1}$$

And, via equations (2a,b) and equivalence principle,  

$$ \frac{L_2}{L_1} = \frac{z_2^2}{z_1^2} = n^2 $$

The series equivalent of the primary winding is:  

$$ U_1(j\omega) = I_1(j\omega)\big(R_1 + X(j\omega)\big) + j\omega MI_2(j\omega) \quad\text{where }\; X(j\omega) = j\omega L_{l1} + X_1(j\omega)$$

$$ B_1(j\omega) = \frac{1}{j\omega L_m} + \frac{1}{j \omega L_1} = -j\,\frac{1}{\omega L_m} \quad\implies\quad X_1(j\omega) = j\omega L_m$$

\begin{equation}\tag{M13a}
    U_1(j\omega) = I_1(j\omega)\big(R_1 + j\omega(L_{l1} + L_m)\big) + j\omega MI_2(j\omega)
\end{equation}

And, the secondary winding,  

$$ U_2(j\omega) = R_2I_2(j\omega) + j\omega L_{l2}I_2(j\omega) + j\omega L_2I_2(j\omega) + j\omega MI_1(j\omega)$$

\begin{equation}\tag{M13b}
    I_2(j\omega) = \frac{U_2(j\omega) - j\omega MI_1(j\omega)}{R_2 + j\omega(L_{l2} + L_2)}
\end{equation}

Via equations (M13a,b) and substitution,  

$$ U_1(j\omega) = I_1(j\omega)\big(R_1 + j\omega(L_{l1}+L_m)\big) + \frac{j\omega MU_2(j\omega) + \omega^2M^2I_1(j\omega)}{R_2 + j\omega(L_{l2} + L_2)}$$

$$ = I_1(j\omega)\big(R_1 + j\omega(L_{l1} + L_m)\big) + \omega M\big(\omega M I_1(j\omega) + jU_2(j\omega)\big)\,\frac{R_2 - j\omega(L_{l2}+L_2)}{R_2^2 + \omega^2(L_{l2}+L_2)^2}$$

$$ U_1(j\omega) = I_1(j\omega)\big(R_1 + j\omega(L_{l1} + L_m)\big)$$

\begin{equation}\tag{M14}
    I_1(j\omega) = \frac{U_1(j\omega)}{R_1 + j\omega(L_{l1} +L_m)}
\end{equation}

Equations (M11a,b) in the frequency domain are:   

\begin{equation}\tag{12a}
    U_1(j\omega) = R_1I_1(j\omega) + j\omega L_{l1}I_1(j\omega) + j\omega L_m\big(I_1(j\omega) + nI_2(j\omega)\big) 
\end{equation}

\begin{equation}\tag{12b}
    U_2(j\omega) = R_2I_2(j\omega) + j\omega L_{l2}I_2(j\omega) + j\omega L_m n\big(I_1(j\omega) + nI_2(j\omega)\big)
\end{equation}

$$ I_2(j\omega) = \frac{U_2(j\omega) - j\omega L_m n I_1(j\omega)}{R_2 + j\omega\big(L_{l2} + L_m n^2\big)}$$

Via the above equation and equation (12a) and substitution,  

$$ U_1(j\omega) = R_1I_1(j\omega) + j\omega L_{l1}I_1(j\omega) + j\omega L_m\left(I_1(j\omega) + \frac{U_2(j\omega) - j\omega L_m n I_1(j\omega)}{R_2 + j\omega\big(L_{l2} + L_m n^2\big)}\right)$$

Multiplying throughout the above equation by the denominator of the identity of $I_2(j\omega)$ gives:  

$$\Big(R_2 + j\omega\big(L_{l2} + L_m n^2\big) \Big)U_1(j\omega) = I_1(j\omega)\Big( R_2 + j\omega\big(L_{l2} + L_m n^2\big)\Big)\big(R_1 + j\omega L_{l1}\big) + j\omega L_mI_1(j\omega)\Big(R_2 + j\omega\big(L_{l2} + L_m n^2\big)\Big) + j\omega L_m\big(U_2(j\omega) - j\omega L_mnI_1(j\omega)\big) $$

$$ I_1(j\omega) = \frac{\Big(R_2 + j\omega\big(L_{l2} + L_m n^2\big) \Big)U_1(j\omega) + j\omega L_mU_2(j\omega)  }{\Big( R_2 + j\omega\big(L_{l2} + L_m n^2\big)\Big)\big(R_1 + j\omega L_{l1}\big) + j\omega L_m\Big(R_2 + j\omega\big(L_{l2} + L_m n^2\big)\Big) + (\omega L_m)^2n}$$

$$ = \frac{R_2U_1(j\omega) + j\omega\Big(U_1(j\omega)\big(L_{l2} + L_mn^2\big) + L_mU_2(j\omega)\Big)}{(\omega L_m)^2n + R_1R_2 - \omega^2L_{l1}\big(L_{l2} + L_mn^2\big) + j\omega\Big(L_{l1}R_2 + R_1\big(L_{l2} + L_mn^2\big) + L_mR_2\Big) - \omega^2L_m\big(L_{l2} + L_mn^2\big)}$$

The KVL equations that describe the circuit in figure 4.7.11 in the frequency domain are:  

$$ U_1(j\omega) = R_1I_1(j\omega) + j\omega L_{l1}I_1(j\omega) + j\omega L_mI_{1a}(j\omega) $$

$$ U_2(j\omega) = R_2I_2(j\omega) + j\omega L_{l2}I_2(j\omega) + j\omega nL_mI_{1a}(j\omega)$$

$$ j\omega L_mI_{1a}(j\omega) = j\omega L_1I_{1b}(j\omega) + j\omega MI_2(j\omega)$$

To recap,  

\begin{equation}\tag{M12}
    \text{a: }\;L_{l1} = \frac{z_1^2}{R_{l1}} \quad\quad \text{b: }\;L_{l2} = \frac{z_2^2}{R_{l2}} \quad\quad \text{c: }\;L_m = \frac{z_1^2}{R_m}
\end{equation}